### ISSUE TITLE
Search in pdf documents doesn’t work

### ISSUE THREAD

**Issue by parsd51** [brave/brave-ios#3312]
### Description: 
Search in pdf documents doesn’t work

### Steps to Reproduce 
  1. Open (just an example): https://www.trm.dk/media/5076/anlaegsstatus-2-halvaar-2020.pdf
  2. Press ... and then Share with..
  3. Press find in page and do a search for an existing text string.

**Actual result:** 
No match is found in the pdf file.

**Expected result:**
Find in page find a matching text string

**Reproduces how often:** [Easily reproduced, Intermittent Issue]
100%

**Brave Version:** 
Brave iPadOS 1.23 (21.2.12.22)

**Device details:** 
iPad Air 2, iPadOS 14.4

**Website problems only:**
- did you check with Brave Shields down? Yes, same problem
- did you check in Safari/Firefox (WkWebView-based browsers)? N/A

**Comment by iccub** [comment]
Fixing for iOS 15+ only with the new api would be acceptable

**Comment by srirambv** [comment]
Verification passed on the following devices running 1.32.5 (21.11.22.20)
- Verified able to search on PDF documents

iPhone 7+ (iOS 14.8) | iPhone XR (iOS 15.1) | iPad Pro (iOS 15.2 Beta 3)
-- | -- | --
<img width=400 src=https://user-images.githubusercontent.com/17010094/143283437-05d98abc-5ae5-4535-ab67-5c12e952ea6d.png> | <img width=320 src=https://user-images.githubusercontent.com/17010094/143283404-17b7d313-7563-4aa1-a90e-85e9062d068a.png> | <img width=560 src=https://user-images.githubusercontent.com/17010094/143283365-b0cd77e6-85a6-4127-88c4-90b38eab0e8b.png>
